src/controllers/SiteController.php

```diff
@@ -226,17 +226,21 @@ public function actionRemoteProceed()
         return $this->redirect(['signup']);
     }
 
-    public function actionSignup()
+    public function actionSignup($scenario = 'default')
     {
         if ($this->user->disableSignup) {
             Yii::$app->session->setFlash('error', Yii::t('hiam', 'Sorry, signup is disabled.'));
 
             return $this->redirect(['login']);
         }
 
+        if ($scenario === 'social') {
```

Comment:
Лучше утащить в константы

```suggestion
        if ($scenario === SignupForm::SCENARIO_SOCIAL) {
```